protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/processinstance/StartInstruction.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.protocol.impl.record.value.processinstance;
+
+import io.camunda.zeebe.msgpack.property.EnumProperty;
+import io.camunda.zeebe.msgpack.property.StringProperty;
+import io.camunda.zeebe.msgpack.value.ObjectValue;
+import io.camunda.zeebe.util.buffer.BufferUtil;
+
+public class StartInstruction extends ObjectValue {
+
+  private final StringProperty elementIdProp = new StringProperty("elementId");
+
+  private final EnumProperty<Instruction> instructionProperty =
+      new EnumProperty<>("instruction", Instruction.class);
```

Comment:
> Out of scope
localVariables (root scope variables are already available)
type (only startBeforeElement will be implemented in the first iteration)
the created element instance keys in the CREATED event (not necessary in first iteration)

The type is still out of scope for the MVP. I see that Nico called it instruction in option 5, but it refers to the same thing.

Reply:
The enum with a single instruction was my interpretation of "only startBeforeElement in the first iteration"

Do you want me to remove it completely?
If no, do you prefer "type" or "instruction"?

Reply:
I would remove it completely. No need to over engineer this now when we can add it later. What are your thoughts @saig0?

If we do decide to keep it I would go with `type` to match the C7 api.

Reply:
@remcowesterhoud I agree with you. Let's remove it for now :+1: We can add it whenever we need it.